snuba/subscriptions/scheduler_processing_strategy.py

```diff
@@ -251,6 +255,17 @@ def submit(self, message: Message[Tick]) -> None:
             self.__next_step.submit(self.__buffers[tick_partition].popleft())
             return
 
+        # Periodically record the legnth of each buffer
+        now = time.time()
+        if now - self.__last_recorded_time > self.__record_frequency_seconds:
+            for partition_index in self.__buffers:
+                self.__metrics.gauge(
+                    "TickBuffer.queue_size",
```

Comment:
DD metrics are usually all lower case and use snake case.
```suggestion
                    "tick_buffer.queue_size",
```